controllers/machine_controller_noderef_test.go

```diff
@@ -123,8 +124,8 @@ func TestGetNode(t *testing.T) {
 			Indexes: []remote.Index{
 				{
 					Object:       &corev1.Node{},
-					Field:        noderefutil.NodeProviderIDIndex,
-					ExtractValue: noderefutil.IndexNodeByProviderID,
+					Field:        index.NodeProviderIDIndex,
+					ExtractValue: index.NodeByProviderID,
 				},
 			},
 		},
```

Comment:
I see we are setting the indexes for the ClusterCacheTracker in several places in the codebase.
What about adding a var named DefaultRemoteIndexes in the new index package (or in the remote package), s in case we should add a new index it is required to change this in a single place?